### DISCUSSION TITLE
Logo for adjacent projects

### DISCUSSION THREAD

**Discussion by tarleb** [jgm/pandoc#9829] [Show and tell]
The old logo that I used for pandoc-adjacent projects, e.g. the [pandoc-ext org](https://github.com/pandoc-ext) and the [@pandoc Mastodon account](https://fosstodon.org/@pandoc), had unclear ownership and licensing, so I needed a new one. This is what I drew up with a bit of SVG yesterday:

![logo](https://raw.githubusercontent.com/tarleb/pandoc-logo/main/pandoc.svg)

Feedback (and  [PRs](https://github.com/tarleb/pandoc-logo)) are very welcome, and feel free to reuse.

**Comment by bpj** [comment]
Well done! I especially like the design suggesting a "document".

What about a version without a background color, perhaps with a frame? I think that would work well as an icon.
